eng/pipelines/weekly_automation.yml

```diff
@@ -0,0 +1,30 @@
+trigger: none
+pr: none
+
+- job: 'RushUpdate'
+  displayName: 'Rush update full'
+  variables:
+  - template: /eng/pipelines/templates/variables/globals.yml
+
+  pool:
+    name: azsdk-pool-mms-ubuntu-2004-general
+    vmImage: MMSUbuntu20.04
+
+  steps:
+  - template: /eng/pipelines/templates/steps/common.yml
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+
+  - script: |
+      node common/scripts/install-run-rush.js install
+      node common/scripts/install-run-rush.js update --full
+    displayName: "Run Rush Update"
+
+  - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
+    parameters:
+      RepoName: azure-sdk-for-js
+      BaseBranchName: $(DefaultBranch)
+      PRBranchName: automated-rush-update-full
+      CommitMsg: "Automatic weekly rush update full"
+      PRTitle: "Automatic weekly rush update full"
```

Comment:
I would recommend using the below title and description

- PR title: `Automatic rush update --full`
- PR description: `This is an automatic PR generated weekly with changes from running the command `rush update --full`

Can we also update the recheck PR title and description similarly?

Reply:
done.